template.yaml

```diff
@@ -44,24 +63,38 @@ Resources:
             Resource: !Sub 'arn:aws:s3:::${S3UploadBucket}/'
             Action:
               - s3:putObjectAcl
+        - Statement:
+          - Effect: Allow
+            Resource: !GetAtt MetaStoreTable.Arn
+            Action:
+            - dynamodb:PutItem
+            - dynamodb:GetItem
+            - dynamodb:Query
+            - dynamodb:Scan
+            - dynamodb:UpdateItem
+            - dynamodb:DeleteItem
       Events:
-        UploadAssetAPI:
+        GetUploadAssetAPI:
           Type: HttpApi
           Properties:
             Path: /uploads
             Method: get
+            ApiId: !Ref MyApi
+        PutUploadAssetAPI:
+          Type: HttpApi
+          Properties:
+            Path: /uploads
+            Method: put
             ApiId: !Ref MyApi  
     Metadata:
       BuildMethod: esbuild
       BuildProperties:
         Format: cjs
         Minify: false
-        Target: "node16"
+        Target: "node18"
         Sourcemap: true
         EntryPoints: 
-          - app.ts
-        External:
-          - aws-sdk
+          - index.js
```

Comment:
I don't remember why external was here before, but maybe it should stay that way. I think it makes the build smaller